<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Param</class>
 <widget class="QDialog" name="Param">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки видеообъекта</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Параметры видеофайла</string>
        </attribute>
        <widget class="QGroupBox" name="groupMedia">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>571</width>
           <height>211</height>
          </rect>
         </property>
         <property name="title">
          <string>Media</string>
         </property>
         <widget class="QLabel" name="labelName">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>141</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Адрес потока видео URL:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="editName">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>90</y>
            <width>351</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>test</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="editPath">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>150</y>
            <width>241</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="editUrl">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>40</y>
            <width>279</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>rtsp://172.254.128.16:8554/1</string>
          </property>
         </widget>
         <widget class="QLabel" name="labelPath">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>90</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Имя файла:</string>
          </property>
         </widget>
         <widget class="QLabel" name="labelUrl">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>150</y>
            <width>141</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Путь сохраняемого файла:</string>
          </property>
         </widget>
         <widget class="QToolButton" name="buttonBrowse">
          <property name="geometry">
           <rect>
            <x>420</x>
            <y>150</y>
            <width>25</width>
            <height>19</height>
           </rect>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Параметры кодека</string>
        </attribute>
        <widget class="QGroupBox" name="groupSettings">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>571</width>
           <height>201</height>
          </rect>
         </property>
         <property name="title">
          <string>Параметры</string>
         </property>
         <widget class="QLabel" name="labelMux">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>40</y>
            <width>59</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Mux</string>
          </property>
         </widget>
         <widget class="QLabel" name="labelVideo">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>90</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Video codec</string>
          </property>
         </widget>
         <widget class="QComboBox" name="comboVideo">
          <property name="geometry">
           <rect>
            <x>188</x>
            <y>100</y>
            <width>171</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QComboBox" name="comboMux">
          <property name="geometry">
           <rect>
            <x>188</x>
            <y>50</y>
            <width>171</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QComboBox" name="comboAudio">
          <property name="geometry">
           <rect>
            <x>188</x>
            <y>150</y>
            <width>171</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="labelAudio">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>150</y>
            <width>59</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Audiocodec</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Таблица</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="2" column="0">
            <widget class="QTableView" name="tableView"/>
           </item>
           <item row="3" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Адрес воспроизведения:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="url">
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>400</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>&lt;null&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="play">
               <property name="text">
                <string>Передать на воспроизведение</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Param</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Param</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
